src/08-time-tracking/timers.apib

```diff
@@ -96,5 +96,6 @@ Update the current timer. Only possible if there is a timer running.
                     + ticket
             + id: `29ff471c-7d8f-40d5-8c95-9a9cab841e65` (string, required)
         + invoiceable: true (boolean, optional)
+        + started_at: `2017-04-26T10:01:49+00:00` (string, optional)
```

Comment:
PP: add it before `description` so the order of the properties is consistent across the endpoints.